core/src/main/scala/org/apache/spark/mapred/SparkHadoopMapRedUtil.scala

```diff
@@ -70,7 +70,8 @@ object SparkHadoopMapRedUtil extends Logging {
       if (shouldCoordinateWithDriver) {
         val outputCommitCoordinator = SparkEnv.get.outputCommitCoordinator
         val taskAttemptNumber = TaskContext.get().attemptNumber()
-        val canCommit = outputCommitCoordinator.canCommit(jobId, splitId, taskAttemptNumber)
```

Comment:
the `jobId` parameter is not used now

Reply:
@cloud-fan yeah, but as I mentioned above, removing jobId from the signature of commitTask will cause a binary incompatibility error, since commitTask here is a public method. So although we no longer use it, we cannot remove it.